utilities/transactions/optimistic_transaction_db_impl.h

```diff
@@ -14,8 +16,15 @@ namespace rocksdb {
 
 class OptimisticTransactionDBImpl : public OptimisticTransactionDB {
  public:
-  explicit OptimisticTransactionDBImpl(DB* db, bool take_ownership = true)
-      : OptimisticTransactionDB(db), db_owner_(take_ownership) {}
+  explicit OptimisticTransactionDBImpl(DB* db, bool take_ownership = true, size_t lock_bucket_size = 1024*1024)
```

Comment:
Formatting.
You can try to run "make format", which will give you instruction about how to install dependencies.